akka-http/src/main/scala/ru/tinkoff/codefest/http/Main.scala

```diff
@@ -17,13 +17,18 @@ object Main extends IOApp {
       FunctionK.lift[IO, Future](unsafeToFuture)
     }
 
+    implicit val nt2: Future ~> IO = {
```

Comment:
How about something like this (using kind-projector)
`λ[Future ~> IO](IO.fromFuture)`
?

Reply:
ok